server/users/tests/routes.test.ts

```diff
@@ -79,5 +79,62 @@ describe("Test users routes", () => {
     );
   });
 
+  // TODO: add content to the body of this test
   test("Correctly updates the cache after user pins board", async function () {});
+
+  test("prevents unauthorized access to the @me Bobadex endpoint", async () => {
+    const res = await request(server.app).get(`/@me/bobadex`);
+    expect(res.status).toBe(401);
+  });
+  
+  test("returns the logged in user's Bobadex data", async () => {
+    setLoggedInUser(JERSEY_DEVIL_USER_ID);
+    const res = await request(server.app).get(`/@me/bobadex`);
+    expect(res.status).toBe(200);
+    expect(res.body).toEqual({
+      seasons: [
+        {
+          id: '9f6d41a5-1e00-4071-9f50-555f1686f87f',
+          name: 'Default',
+          realm_id: 'v0',
```

Comment:
Oooohooh! This shouldn't be v0, it should be the `external_id` of the realm (so a UUID). Also `identities_count` should probably be a number.

I don't think you should fix it as part of this PR because it also implies changes to the frontend, but add a TODO to fix them! It'll surely get caught as part of me getting `zod` everywhere.